media/css/mozorg/home/home-en.scss

```diff
@@ -72,6 +72,22 @@ $image-path: '/media/protocol/img';
     }
 }
 
+//* -------------------------------------------------------------------------- */
+// Pocket Hero
+
+.pocket-sunny-hero {
+    &.mzp-t-product-firefox .mzp-c-hero-title {
```

Comment:
Commandeering this class.  I didn't see one for Pocket, so I overwrote this one. LMK if there's a better way to manage this.

Reply:
👍